src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.scss

```
@@ -1,67 +1,52 @@
-.alerts {
-  height: 17rem;
+.details {
```

Rather than creating a new style, why not reaching the DOM through a specific CSS selector:
```suggestion
cd-card[cardTitle="Details"] {
```

---

Generally, I never liked the idea to use specific selector which are tags rather to use specific class names to do  specific stuff. Like in this case we might change the title of card at any point and that will affect the styling of the page and developer might always be confused why this happened.

---

You're right, that attribute is not the best one! In any case, I'd rather add an `id="details-card"` (or something similar) to the DOM element than a new CSS class. IMHO CSS classes should be reusable and applicable to multiple elements, while for unique (semantic) items we should tag them with IDs.